packages/svelte/src/internal/client/reactivity/store.js

```diff
@@ -32,7 +31,16 @@ export function store_get(store, store_name, stores) {
 			set(entry.source, undefined);
 			entry.unsubscribe = noop;
 		} else {
-			entry.unsubscribe = subscribe_to_store(store, (v) => set(entry.source, v));
+			var initial = true;
+
+			entry.unsubscribe = subscribe_to_store(store, (v) => {
```

Comment:
should have a comment explaining why we need to do this

Reply:
done